### PR TITLE
Show dot if day has overdue tasks

### PR DESCRIPTION
**Pull request by sebastiantoh**
https://github.com/0dy553y/odyssey/issues/112

image

~~Not sure if we still want to show the dot if the date is currently the selected date. Thoughts?~~

Nvm, i'll hide it if the date is currently selected otherwise it looks ugly on mobile

Also, fixes https://github.com/0dy553y/odyssey/issues/252 by disabling horizontal panning in user task description
- To reproduce original issue, try scrolling the user task card in mobile, by dragging from the task description

### CODE REVIEW

File: frontend/src/components/home/UserTaskCard.tsx [hunk 1]
```diff
@@ -56,6 +56,10 @@ const useStyles = makeStyles(() => ({
     // Padding for scrollbar
     paddingRight: '8px',
     overflowY: 'auto',
+    overflowX: 'hidden',
+    // Only allow panning vertical. Specifically, disallow panning horizontal
```
Comment (sebastiantoh) [inline frontend/src/components/home/UserTaskCard.tsx:60]:
```suggestion
    // Only allow panning vertically. Specifically, disallow panning horizontally
```

**Review by ianyong — COMMENTED** [review]
The dots are cut off on certain resolutions.

image

**Comment by sebastiantoh** [comment]
ty, should be fixed, i hope! 

also, reverted back to show dot even if date is selected. Felt it was quite disconcerting to see the dot disappear

**Review by ianyong — APPROVED** [review]
LGTM!

**Pull request merged**